README.md

```diff
@@ -1,568 +1,183 @@
-
-```python
-AIConfigRuntime.register_model_parser(myModelParserInstance, ["gpt-4"])
+config.add_prompt("gen_packing_list", gen_packing_list)
+packing_list = await config.run("gen_packing_list", options=inference_options)
 ```
 
-### Callback events
-
-Use callback events to trace and monitor what's going on -- helpful for debugging and observability.
-
-```python
-from aiconfig import AIConfigRuntime, CallbackEvent, CallbackManager
-config = AIConfigRuntime.load('aiconfig.json')
-
-async def my_custom_callback(event: CallbackEvent) -> None:
-  print(f"Event triggered: {event.name}", event)
-
-callback_manager = CallbackManager([my_custom_callback])
-config.set_callback_manager(callback_manager)
-
-await config.run("prompt_name")
-```
-
-[**Read more** here](https://aiconfig.lastmileai.dev/docs/overview/monitoring-aiconfig)
-
-## Extensibility
-
-AIConfig is designed to be customized and extended for your use-case. The [Extensibility](/docs/extensibility) guide goes into more detail.
-
-Currently, there are 3 core ways to extend AIConfig:
-
-1. [Supporting other models](https://aiconfig.lastmileai.dev/docs/extensibility#1-bring-your-own-model) - define a ModelParser extension
-2. [Callback event handlers](https://aiconfig.lastmileai.dev/docs/extensibility#2-callback-handlers) - tracing and monitoring
-3. [Custom metadata](https://aiconfig.lastmileai.dev/docs/extensibility#3-custom-metadata) - save custom fields in `aiconfig`
-
-## Contributing to `aiconfig`
-
-This is our first open-source project and we'd love your help.
-
-See our [contributing guidelines](https://aiconfig.lastmileai.dev/docs/contributing) -- we would especially love help adding support for additional models that the community wants.
-
-## Cookbooks
-
-We provide several guides to demonstrate the power of `aiconfig`.
-
-> **See the [`cookbooks`](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks) folder for examples to clone.**
-
-### Chatbot
+## Features
 
-- [Wizard GPT](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Wizard-GPT) - speak to a wizard on your CLI
+- **Prompts as configs**: a [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt chains, model settings, and flexible metadata in source control.
+- **Editor for Prompt Chains**: Quickly prototype and iterate on your prompt chains and model settings in [AI Workbooks](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9), a notebook-like playground for generative AI.
+- **Model-agnostic SDK**: Access the latest models like gpt-4, Gemini, Mistral models, and HuggingFace models. Easily extensible to work with any generative AI model.
+- **Multimodal Capabilities**: Design multimodal generative AI apps that chain text, image, and audio models together.
+- **Collaboration for Teams**: Improved collaboration between devs and prompt engineers given clear abstractions of generative AI components and application code.
```

Comment:
I don't like the wording of this. "devs" and "prompt engineers" are not usually different. Better framing would be:

```
**More collaborative**: With an aiconfig, different people on your team can work on prompt engineering, app development, and collaborate using this version controlled artifact for generative AI..
```